pkg/reconciler/v1alpha1/taskrun/resources/apply.go

```diff
@@ -43,24 +43,14 @@ func ApplyParameters(spec *v1alpha1.TaskSpec, tr *v1alpha1.TaskRun, defaults ...
 
 // ApplyResources applies the templating from values in resources which are referenced in spec as subitems
 // of the replacementStr. It retrieves the referenced resources via the getter.
```

Comment:
i think the comment here is not quite accurate anymore

Reply:
Removed!